pypy/doc/conf.py

```diff
@@ -36,6 +36,13 @@
         print('sphinx_rtd_theme is not installed')
         html_theme = 'default'
 
+# Set canonical URL from the Read the Docs Domain
+html_baseurl = os.environ.get("READTHEDOCS_CANONICAL_URL", "")
+
+# Tell Jinja2 templates the build is running on Read the Docs
+if os.environ.get("READTHEDOCS", "") == "True":
+    html_context["READTHEDOCS"] = True
```

Comment:
They should fix this in the copy paste email, but you need `html_context = {}` above.

Reply:
They fixed it in the post, and new emails will have this snippet as well
https://github.com/readthedocs/website/pull/314

Reply:
Ah nice!